lib/buildFactory.js

```diff
@@ -295,6 +295,72 @@ class BuildFactory extends BaseFactory {
         return super.list(config);
     }
 
+    /**
+     * Get build statuses for jobs
+     * @method getBuildStatuses
+     * @param  {Object}   config                  Config object
+     * @param  {Array}    config.jobIds           Jobs to get build statuses for
+     * @param  {Number}   config.offset           Number of build statuses to skip
+     * @param  {Number}   config.numBuilds        Number of build statuses to return per job
+     * @return {Promise}
+     */
+    getBuildStatuses(config) {
+        config.offset = config.offset || 0;
+        config.numBuilds = config.numBuilds || 1;
+        config.jobIds = config.jobIds || [];
+
+        const query =
```

Comment:
Use string literals instead of concatenating the query

Reply:
Also, thinking would it make sense to have a separate file for raw queries which can be required here. Would be easy to compare when we do several changes over the time and also more maintainable.

Reply:
If I do a string literal the string exceeds the maximum lint line length, when I tried to do a multi line string I got another lint issue about multi line strings only being supported in es6+.

Is there another way to do this or should I disable the linter for the string?